tests/Tests/400__ods_dashboard/400__ods_dashboard.robot

```diff
@@ -115,6 +116,24 @@ Search and Verify GPU Items Appears In Resources Page
     Search Items In Resources Section   GPU
     Check GPU Resources
 
+Verify Notifications Are Shown When Notebook Builds Have Not Started
+    [Documentation]     Verifies that Notifications are shown in RHODS Dashboard when Notebook builds haven't started
+    [Tags]    Sanity
+    ...       ODS-1347
+    ${builds_to_be_deleted}=  Create List  pytorch  tensorflow  minimal  cuda-s2i-thoth
```

Comment:
could you move the variable definitions (line 123 and 125) to `*** Variables ***` section?

Reply:
Updated.

Reply:
I don't see this change

Reply:
Updated